Sources/ViewControllers/Settings/CreditsViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CreditsViewController.swift
+//  OpenFoodFacts
+//
+//  Created by Егор on 2/16/18.
+//  Copyright © 2018 Andrés Pizá Bückmann. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
```

Comment:
import SnapKit (you'll need to merge master before, I've updated the dependencies)